packages/@aws-cdk/aws-glue-alpha/lib/connection.ts

```diff
@@ -96,21 +156,76 @@ export class ConnectionType {
    */
   public static readonly JIRACLOUD = new ConnectionType('JIRACLOUD');
 
+  /**
+   * Designates a connection to LinkedIn.
+   */
+  public static readonly LINKEDIN = new ConnectionType('LINKEDIN');
+
+  /**
+   * Designates a connection to Mailchimp.
+   */
+  public static readonly MAILCHIMP = new ConnectionType('MAILCHIMP');
+
   /**
    * Designates a connection to Adobe Marketo Engage.
    */
   public static readonly MARKETO = new ConnectionType('MARKETO');
 
+  /**
+   * Designates a connection to Microsoft Dynamics 365 Finance and Operations.
+   */
+  public static readonly MICROSOFTDYNAMIC365FINANCEANDOPS = new ConnectionType('MICROSOFTDYNAMIC365FINANCEANDOPS');
+
+  /**
+   * Designates a connection to Microsoft Teams.
+   */
+  public static readonly MICROSOFTTEAMS = new ConnectionType('MICROSOFTTEAMS');
+
+  /**
+   * Designates a connection to Mixpanel.
+   */
+  public static readonly MIXPANEL = new ConnectionType('MIXPANEL');
+
   /**
    * Designates a connection to Oracle NetSuite.
    */
   public static readonly NETSUITEERP = new ConnectionType('NETSUITEERP');
 
+  /**
+   * Designates a connection to PayPal.
+   */
+  public static readonly PAYPAL = new ConnectionType('PAYPAL');
+
+  /**
+   * Designates a connection to Pendo.
+   */
+  public static readonly PENDO = new ConnectionType('PENDO');
+
+  /**
+   * Designates a connection to Pipedrive.
+   */
+  public static readonly PIPEDIVE = new ConnectionType('PIPEDIVE');
```

Comment:
Is this a typo? Should probably be `PIPEDRIVE`.

Reply:
all fixed